app/src/main/java/org/gdg/frisbee/android/event/EventOverviewFragment.java

```diff
@@ -191,6 +191,10 @@ public void failure(RetrofitError error) {
     }
 
     private void updateGroupDetails(Chapter group) {
+        if (getActivity() == null) {
```

Comment:
The problem is with Fragments in general. Since there is an async operation going on, when user press back before the operation completes, Fragment looses its tie with the Activity. Other option to solve this is to cancel all async operations in onDestroy